app/code/community/Nexcessnet/Alarmbell/Helper/Data.php

```diff
@@ -18,74 +18,87 @@
  * with this program; if not, write to the Free Software Foundation, Inc.,
  * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  */
-
-class Nexcessnet_Alarmbell_Helper_Data extends Mage_Core_Helper_Abstract {
-
-	/**
-     * Formats and logs a message through Magento's logging facility
+class Nexcessnet_Alarmbell_Helper_Data extends Mage_Core_Helper_Abstract
+{
+    /**
+     * Formats and logs a message through Magento's logging facility.
+     *
+     * @param string $message
      *
-     * @param  string $message
+     * @return string
      */
-    function log($message) {
-    	// get the client IP
-		$remoteIp = Mage::helper('core/http')->getRemoteAddr();
+    public function log($message)
+    {
+        // get the client IP
+        $remoteIp = Mage::helper('core/http')->getRemoteAddr();
 
-		// get user's admin username (if logged-in)
-		$adminUsername = '';
-		$admin = Mage::getSingleton('admin/session')->getUser();
-        if (is_object($admin)) { 
-            if ($admin->getId()) { $adminUsername = $admin->getUsername(); }
+        // get user's admin username (if logged-in)
+        $adminUsername = '';
+        $admin = Mage::getSingleton('admin/session')->getUser();
+        if (is_object($admin)) {
+            if ($admin->getId()) {
+                $adminUsername = $admin->getUsername();
+            }
+        } else {
+            $adminusername = '';
         }
-        else { $adminusername = ''; }
 
         // construct log message, and log it
-        $logMessage = 'ALARMBELL ('. $remoteIp . ')';
-        if (!empty($adminUsername)) { $logMessage .= " [$adminUsername]"; }
-        $logMessage .= ': ' . $message;
+        $logMessage = 'ALARMBELL ('.$remoteIp.')';
+        if (!empty($adminUsername)) {
+            $logMessage .= " [$adminUsername]";
+        }
+        $logMessage .= ': '.$message;
```

Comment:
This change actually moves further from the Zend guidelines, should stay as `': ' . $message`